packages/shared/src/components/notifications/NotificationItemAvatar.tsx

```diff
@@ -18,7 +20,16 @@ function NotificationItemAvatar({
 
   if (type === NotificationAvatarType.User) {
     return (
-      <ProfilePicture user={{ image, id: referenceId }} nativeLazyLoading />
+      <ProfileTooltip
+        link={{ href: `${webappUrl}/${referenceId}` }}
```

Comment:
Avatar has the property `targetUrl` please use it instead

Reply:
Oh, yep. That is much better. Let me change this.